packages/epo-react-lib/.storybook/i18next.js

```diff
@@ -0,0 +1,24 @@
+import { initReactI18next } from "react-i18next";
```

Comment:
this is an i18next setup for Storybook to render locale strings, there is also a Storybook addon that lets you change locale in browser by clicking a little globe icon